src/nwb_conversion_tools/utils/checks.py

```diff
@@ -2,6 +2,9 @@
 
 
 def check_regular_series(series: np.ndarray, tolerance_decimals: int = 9):
-    """General purpose function for checking if the difference between all consecutive points in a series are equal."""
-    uniq_diff_ts = np.unique(np.diff(series).round(decimals=tolerance_decimals))
-    return len(uniq_diff_ts) == 1
+    """General purpose function for checking if the difference between all consecutive points in a series are equal.
+    If it is, returns the assessed sampling rate."""
+    diff_ts = np.diff(series).round(decimals=tolerance_decimals)
+    uniq_diff_ts = np.unique(diff_ts)
+    rate = diff_ts[0] if len(uniq_diff_ts) == 1 else None
+    return len(uniq_diff_ts) == 1, rate
```

Comment:
@h-mayorquin FYI this kind of thing is exactly why I wanted all of these util functions centralized somewhere - we had decided the NWBInspector was a good place to centralize such things since it's secondary dependencies are extremely minimal on top of PyNWB.

This type of added functionality would be useful to have over there, but now we have to both remember to update it in both places, which is a very easy thing to overlook, and takes twice as much work to integrate.

Would it bother you overly much if we just had a separate repo altogether, like `cn_utils`, that both the NWBInspector and NWBCT can both depend on?